routes/message.js

```diff
@@ -64,7 +64,7 @@ router.ws("/",function(ws,req){
                 case TEXT_COMMUNICATION:
                 {    
                     let msg_id;
-                    delete msg.type
+                    
                     if(!msg.text){
                         ws.send('Invalid body');
                         break;
```

Comment:
`ws.onclose` in line 97 needs to be updated from `username` to `phone`. Without that the user is not deleted from the list of clients if he exits and message is still sent on line 74